src/components/header/NavButton/NavButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import classnames from 'classnames'
+
+export const NavButton = (
+  props: React.HTMLAttributes<HTMLButtonElement>
+): React.ReactElement => {
+  const { children, onClick, className, ...buttonProps } = props
+
+  const classes = classnames(className)
```

Comment:
I removed classNames. The reason I created the NavButton component is because <Button> always includes the `usa-button` class. None of the nav buttons have this class on them, which is why I don't think <Button> should be used here. 

Here is the scss for the header components if you would like to verify :) https://github.com/uswds/uswds/blob/develop/src/stylesheets/components/_header.scss (or if I am missing something)

Reply:
Make sense!